docs/001_develop/03_client-capabilities/008_interaction/text-area.mdx

```diff
@@ -112,23 +110,170 @@ Use cases:
-| rows        | `number`  | Defines the size of the element vertically. **Default: 2**                                                  | |
-| value       | `string`  | The text area content                                                                                       | `<rapid-text-area :value=${sync((x) => x.text_area_value)}>` |
+### Attributes:
+
+<table>
+  <thead><tr><th>Name</th><th>Type</th><th>Description</th><th>Example</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>appearance</td>
+      <td><code>string</code></td>
+      <td>Changes the general view of the element; this can be **filled** or **outline**</td>
+<td>
+
+```typescript
+<text-area appearance="filled">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>autofocus</td>
+      <td><code>boolean</code></td>
+      <td>When true, the component will be focused when the page has finished loading</td>
+<td>
+
+```typescript
+<text-area autofocus>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>cols</td>
+      <td><code>number</code></td>
+      <td>Defines the size of the element horizontally. **Default: 20**</td>
+<td>
+
+```typescript
+<text-area cols=2>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>disabled</td>
+      <td><code>boolean</code></td>
+      <td>Similar to `readonly`, but with a blur on the component</td>
+<td>
+
+```typescript
+<text-area disabled>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>formId</td>
+      <td><code>string</code></td>
+      <td>Associates this component to a form. Form `id` needs to be passed. If no Id informed, then it will be associated with the ancestor form</td>
+<td>
+
+```typescript
+<text-area formId="formId">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>maxlength</td>
+      <td><code>number</code></td>
+      <td>The maximum number of characters allowed</td>
+<td>
+
+```typescript
+<text-area maxlength=100>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>minlength</td>
+      <td><code>number</code></td>
+      <td>The minimum number of characters allowed</td>
+<td>
+
+```typescript
+<text-area minlength=50>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>name</td>
+      <td><code>string</code></td>
+      <td>Defines the name of the element</td>
+<td>
+
+```typescript
+<text-area name="textAreaName">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>placeholder</td>
+      <td><code>string</code></td>
+      <td>Sets placeholder text for the element, which disappears when the user starts inputting</td>
+<td>
+
+```typescript
+<text-area placeholder="Sample place holder text">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>readonly</td>
+      <td><code>boolean</code></td>
+      <td>If true, the user cannot change the value of this field</td>
+<td>
+
+```typescript
+<text-area readonly>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>resize</td>
+      <td><code>string</code></td>
+      <td>Allows the user to resize the element; this can be {"none","both","vertical","horizontal"}. **Default: none**</td>
```

Comment:
This is the same issue I had, the `{ ... }` syntax you're using doesn't render correctly on docusouras. 
```suggestion
      <td><code>"none" | "both" | "vertical" | "horizontal"</code></td>
      <td>Configures which ways the user can resize the text area. **Default: none**</td>
```

![Screenshot 2024-11-18 at 08 39 15](https://github.com/user-attachments/assets/1426b010-61e6-44b5-834e-871109d7ec0f)